src/configuration.js

```diff
@@ -107,7 +107,7 @@ export class Configuration {
         defaultDatasetId: LOCAL_ENV_VARS[ENV_VARS.DEFAULT_DATASET_ID],
         defaultRequestQueueId: LOCAL_ENV_VARS[ENV_VARS.DEFAULT_REQUEST_QUEUE_ID],
         inputKey: 'INPUT',
-        apiBaseUrl: 'https://api.apify.com/v2',
+        apiBaseUrl: 'https://api.apify.com/',
```

Comment:
if we had `/v2/v2` there now, we should skip the trailing slash too, otherwise we end up with `//v2`
```suggestion
        apiBaseUrl: 'https://api.apify.com',
```